custom-docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM trinodb/trino:472
+#FROM trinodb/trino:${TRINO_VERSION}
+
+# make it more like the core container for now
+RUN rm -rf /etc/trino/catalog
+RUN rm -rf /usr/lib/trino/plugin/phoenix
```

Comment:
This doesn't make the image smaller, unless the layers are squashed. I guess the only advantage of removing plugins this way is to avoid loading them on startup, but I'd wait for the core image to be released.